lldb/source/Symbol/TypeSystem.cpp

```diff
@@ -337,4 +346,4 @@ void TypeSystemMap::RemoveTypeSystemsForLanguage(lldb::LanguageType language) {
       m_map.erase(pos);
   }
 }
-
+// END APPLE
```

Comment:
We have been using // BEGIN SWIFT & // END SWIFT, not APPLE.  Was there a reason this was switched to APPLE, or just an oversight.  It would be better to be consistent with these markers.

Reply:
Just an oversight, not sure why I chose apple instead of swift 🤔. I can switch it back, though if you think it's not used anywhere I'm happy to just remove it entirely.

Reply:
There's a use in `lldb/source/Target/Target.cpp:GetScratchTypeSystemForLanguage` which is used in `lldb/source/Plugins/ExpressionParser/Swift/SwiftREPL.cpp:CreateInstanceFromTarget`. So maybe can't remove?

Reply:
Eh, you are right, we allow people to pass compiler options to the REPL compiler.  For now we need this.

Reply:
Okay, thanks for having a look :)! I'll merge this now then and cherry-pick that change to `next` as well.